### PR TITLE
Readme: add F# example for verbs

### PR DESCRIPTION
**Pull request by ForNeVeR**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -136,7 +136,8 @@ class CloneOptions {
 }
 
 int Main(string[] args) {
-  return CommandLine.Parser.Default.ParseArguments<AddOptions, CommitOptions, CloneOptions>(args)
```
Comment (mizipzor) [inline README.md]: I suggest leaving this as is, could we limit this PR to adding the F# example? Feel free to force push if you want clean history.

Comment (ForNeVeR) [inline README.md]: I am sorry, but why? Current example is _invalid_ as far as I can see.

Comment (ForNeVeR) [inline README.md]: There is no `ParseArguments` variant with more that 1 generic parameter. That's the reason I changed that line.

Comment (ForNeVeR) [inline README.md]: Ah, sorry, ignore that. It was an extension method. Okay, I will clean that up, thanks for the suggestion.

**Comment by ForNeVeR** [comment]
Fixed the PR.

**Comment by mizipzor** [comment]
Good work, thank you.

**Pull request merged**